### DISCUSSION TITLE
error - ./src/.umi/plugin-layout/Layout.tsx:119:30-39 export 'ProLayout' (imported as 'ProLayout') was not found in 'xx/node_modules/@ant-design/pro-layout'

### DISCUSSION THREAD

**Discussion by yuanliu147** [umijs/umi#11608] [Q&A]
error - ./src/.umi/plugin-layout/Layout.tsx:119:30-39
export 'ProLayout' (imported as 'ProLayout') was not found in 'xxx/node_modules/@ant-design/pro-layout'

**Comment by yuanliu147** [comment]
为什么 每次运行项目就会出现这种？不应该是 import ProLayout from '../xxxx' 么，，，

**Comment by fz6m** [comment]
可以把 node_modules 删了再安装一遍，可能没安装好。

如有问题，需要给一个最小复现。
